test/tests/ver-1fb/comparison_ver-1fb.py

```diff
@@ -26,28 +26,28 @@ def summation_terms(n, x, t, alph):
 
```

Comment:
Please update this script so that the analytical solution is plotted with a line rather than scatter marks. Since the analytical solution is continuous, it should be plot with a continuous line.